lib/FusionInventory/Agent/Task/Inventory/Win32/Crontasks.pm

```diff
@@ -0,0 +1,239 @@
+                    my @wdays;
+                    my $daysOfMonth = $trigger->findnodes('ScheduleByMonth/DaysOfMonth');
+                    foreach my $day ($daysOfMonth->get_nodelist) {
+                        push(@wdays, $day->string_value);
+                    }
+                    $executionDay = join(",", @wdays);
+                }
+                if ($trigger->find('ScheduleByMonth/Months')) {
+                    my @months;
+                    my $byMonths = $trigger->findnodes('ScheduleByMonth/Months');
+                    if ($trigger->find('ScheduleByMonth/Months/January')) {
+                        push(@months, 1);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/February')) {
+                        push(@months, 2);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/March')) {
+                        push(@months, 3);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/April')) {
+                        push(@months, 4);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/May')) {
+                        push(@months, 5);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/June')) {
+                        push(@months, 6);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/July')) {
+                        push(@months, 7);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/August')) {
+                        push(@months, 8);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/September')) {
+                        push(@months, 9);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/October')) {
+                        push(@months, 10);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/November')) {
+                        push(@months, 11);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/December')) {
+                        push(@months, 12);
+                    }
+                    $executionMonth = join(",", @months);
+                }
+            }
+        }
+        # Replace */1 in * (more simple to read)
+        if ($executionMonth eq '*/1') {
+            $executionMonth = '*';
+        }
+        if ($executionDay eq '*/1') {
+            $executionDay = '*';
+        }
+        if ($executionHour eq '*/1') {
+            $executionHour = '*';
+        }
+        if ($executionMinute eq '*/1') {
+            $executionMinute = '*';
+        }
+        if ($executionWeekday eq '*/1') {
+            $executionWeekday = '*';
+        }
+        my $status = 1;
+        if ($xp->find('/Task/Settings/Enabled')->string_value ne 'true') {
+            $status = 0;
+        }
```

Comment:
Safer as nothing tell you the string_value call returns a defined value:
```suggestion
        my $enabled = $xp->find('/Task/Settings/Enabled')->string_value;
        my $status = $enabled && $enabled eq 'true' ? 1 : 0;
```
The $status evaluation can also directly be integrated l.233.